plugins/languages/texpresso.nix

```diff
@@ -0,0 +1,16 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.vim-plugin.mkVimPlugin config {
```

Comment:
Would be nice to have a comment stating that the plugin has no options to configure